integration-tests/src/test/java/io/unitycatalog/integrationtests/SparkDeltaTableCRUDTest.java

```diff
@@ -10,126 +14,120 @@
-                .as("Data after CREATE should be empty")
-                .isEqualTo(List.of());
-    }
-
-    @ParameterizedTest
-    @MethodSource("locationTypes")
-    @Order(2)
-    public void insert(LocationType locationType) {
-        String table = getTableName(locationType);
-        spark.sql(format("INSERT INTO %s VALUES (0, 0), (1, 1.5), (42, 244.25), (539, 425.66102859000944)", table));
-
-        assertThat(getData(table))
-                .as("Data after INSERT")
-                .isEqualTo(List.of(
-                        "{\"as_int\":0,\"as_double\":0.0}",
-                        "{\"as_int\":1,\"as_double\":1.5}",
-                        "{\"as_int\":42,\"as_double\":244.25}",
-                        "{\"as_int\":539,\"as_double\":425.66102859000944}"
-                ));
-    }
-
-    @ParameterizedTest
-    @MethodSource("locationTypes")
-    @Order(3)
-    public void merge(LocationType locationType) {
-        String table = getTableName(locationType);
-        spark.sql(format("""
-                MERGE INTO %s target
-                USING (SELECT * FROM VALUES (1, 1.001), (3, 3.75), (5, 2.5) AS mock_data(as_int, as_double)) source
-                ON source.as_int = target.as_int
-                WHEN MATCHED THEN DELETE
-                WHEN NOT MATCHED THEN INSERT *
-                """, table));
-
-        assertThat(getData(table))
-                .as("Data after MERGE")
-                .isEqualTo(List.of(
-                        "{\"as_int\":0,\"as_double\":0.0}",
-                        "{\"as_int\":3,\"as_double\":3.75}",
-                        "{\"as_int\":5,\"as_double\":2.5}",
-                        "{\"as_int\":42,\"as_double\":244.25}",
-                        "{\"as_int\":539,\"as_double\":425.66102859000944}"
-                ));
-    }
-
-
-    @ParameterizedTest
-    @MethodSource("locationTypes")
-    @Order(4)
-    public void delete(LocationType locationType) {
-        String table = getTableName(locationType);
-        spark.sql(format("DELETE FROM %s WHERE as_int < 20", table));
-
-        assertThat(getData(table))
-                .as("Data after DELETE")
-                .isEqualTo(List.of(
-                        "{\"as_int\":42,\"as_double\":244.25}",
-                        "{\"as_int\":539,\"as_double\":425.66102859000944}"
-                ));
-    }
-
-    @ParameterizedTest
-    @MethodSource("locationTypes")
-    @Order(5)
-    public void update(LocationType locationType) {
-        String table = getTableName(locationType);
-        spark.sql(format("UPDATE %s SET as_double = as_int * 1.5 WHERE as_int = 42", table));
-
-        assertThat(getData(table))
-                .as("Data after UPDATE")
-                .isEqualTo(List.of(
-                        "{\"as_int\":42,\"as_double\":63.0}",
-                        "{\"as_int\":539,\"as_double\":425.66102859000944}"
-                ));
-    }
+  private static final String BASE_TABLE_NAME = "numbers";
```

Comment:
This file is similar to the comment, just reformat the code to fix the checkstyle report.
https://github.com/unitycatalog/unitycatalog/pull/1119/files#r2480136528